lib/logflare_grpc/trace/server.ex

```diff
@@ -13,15 +13,21 @@ defmodule LogflareGrpc.Trace.Server do
 
   require Logger
 
-  def export(%{resource_spans: resource_spans}, stream) do
+  def export(events, stream) do
     with %{"x-api-key" => api_key, "x-source-id" => source_token} <-
            GRPC.Stream.get_headers(stream),
          user when not is_nil(user) <- Users.get_by(api_key: api_key),
```

Comment:
should not be using the `api_key` field, should abtstract and/or reuse out the logic in the VerifyApiAccess plug so that we can use both access tokens as well as api_key field.